### ISSUE TITLE
Incorrect method reference in docs

### ISSUE THREAD

**Issue by amoeba** [pjamesjoyce/lcopt#6]
Hi @pjamesjoyce, I'm starting to go through this repo for https://github.com/openjournals/joss-reviews/issues/339 and noticed what appears to be a typo in a method reference in the docs:

In the FORWAST setup instructions, http://lcopt.readthedocs.io/en/latest/1_installation.html#option-1-run-the-setup-utility-in-a-jupyter-notebook-python-shell, I see a reference to:

```py
from lcopt.utils import lcopt_bw2_setup_forwast
lcopt_bw2_setup_forwast()
```

which doesn't work:

```
ImportError: cannot import name 'lcopt_bw2_setup_forwast'
```

But in the script mentioned in Option 2 I see:

```py
from lcopt.utils import lcopt_bw2_forwast_setup

if __name__ == "__main__":
    lcopt_bw2_forwast_setup()
```

which works. Note the slight naming difference. Is Option 1 just wrong?

**Comment by pjamesjoyce** [comment]
Hi @amoeba , you're quite right. It was a typo in the docs, apologies.

It's fixed in the latest commit (c627640edb8e999acf85b3ed0ba73506faeeb705)

` lcopt_bw2_forwast_setup()` is the correct version

**Comment by amoeba** [comment]
Thanks!
